index.js

```diff
@@ -46,7 +49,7 @@ function commandResponseReceived( id, blog, subscriptionMessage ) {
 xmpp.registerCommandResponseCallBack( commandResponseReceived );
 
 function blogPath( blogUrl ) {
-	const urlParts = url.parse( blogUrl );
+	const urlParts = url.parse( `http://${blogUrl}` );
```

Comment:
I think we could allow a blog address with and without the scheme here.
In any case, the jabber sub command works with many types of input so we don't need much preprocessing here in theory.